filament/backend/src/metal/MetalHandles.mm

```diff
@@ -275,8 +278,7 @@ BufferDescriptor b(nullptr, 0u, [](void* buffer, size_t size, void* user) {
     : HwVertexBuffer(bufferCount, attributeCount, vertexCount, attributes), buffers(bufferCount, nullptr) {}
 
 MetalIndexBuffer::MetalIndexBuffer(MetalContext& context, BufferUsage usage, uint8_t elementSize,
-        uint32_t indexCount) : HwIndexBuffer(elementSize, indexCount),
-        buffer(context, usage, elementSize * indexCount, true) { }
+        uint32_t indexCount) : HwIndexBuffer(elementSize, indexCount) { }
```

Comment:
Where did this initialization of `buffer` go? Also, if the intention is to keep `buffer` empty at this point, it must be set to `nullptr` at least.

Reply:
`buffer` is just a reference now to a `MetalBuffer` which owned by a `BufferObject` (previosly `MetalIndexBuffer` owned this `MetalBuffer`). I agree, it should be a nullptr by default.

Reply:
fixed in https://github.com/shapr3d/filament/pull/89/commits/043bd4662f4a53b9fc2cc4f54987d676365609d8